### ISSUE TITLE
Delay when pressing

### ISSUE THREAD

**Issue by Ricardo1980** [KhaosT/homebridge-amazondash#2]
Hello!
Is there a delay of several seconds between when I press the button and the action is triggered?
Thanks.

**Comment by KhaosT** [comment]
Yeah, there is. The delay is caused by Amazon Dash trying to join the WiFi network.

**Comment by Ricardo1980** [comment]
I guess you can improve that using DHCP Reservation in your router

**Comment by TerenceTanWT** [comment]
It takes at least 10 seconds for my scene to be triggered after pressing the button. Maybe you can consider using [this technique](https://github.com/maxisme/amazon-bash), which aims to reduce the response time to about 2-3 seconds without Wi-Fi required.
